src/script/error/change.js

```diff
@@ -0,0 +1,16 @@
+// 테두리 색 바꾸기
+export function changeBorder(target, color = "red") {
+    target.style.border = `1px solid var(--Linkbrary-${color})`;
```

Comment:
color에 color변수에 해당하지 않는 값이 들어오면 에러가 나겠네요. 확장성을 고려해 아래처럼 바꾸면 좋겠습니다.
아니면 매개변수명에서 color에 어떤 값을 보내야하는지 힌트를 줘야 좋을 것 같아요.

```suggestion
    target.style.border = `1px solid ${color}`;
```